<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SendEmailDlg</class>
 <widget class="QDialog" name="SendEmailDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>691</width>
    <height>573</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Envio de correo electrónico</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/resources.qrc">
    <normaloff>:/aplicacion/images/alephLogoIcon.ico</normaloff>:/aplicacion/images/alephLogoIcon.ico</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0" colspan="2">
    <widget class="QGroupBox" name="gbMain">
     <property name="title">
      <string>Destinatario</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <widget class="QLabel" name="label">
        <property name="text">
         <string>Responder a:</string>
        </property>
       </widget>
      </item>
      <item row="0" column="2" colspan="2">
       <widget class="QLineEdit" name="txtFrom"/>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lblTo">
        <property name="text">
         <string>Para:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <widget class="QPushButton" name="pbContacts">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="toolTip">
         <string>Agenda de contactos</string>
        </property>
        <property name="statusTip">
         <string>Agenda de contactos</string>
        </property>
        <property name="whatsThis">
         <string>Agenda de contactos</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../resources/resources.qrc">
          <normaloff>:/generales/images/addressbook.png</normaloff>:/generales/images/addressbook.png</iconset>
        </property>
        <property name="shortcut">
         <string>Alt+C</string>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="label_3">
        <property name="text">
         <string>Copia a:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="2">
       <widget class="QLineEdit" name="txtCopy"/>
      </item>
      <item row="2" column="3">
       <widget class="QPushButton" name="pbCopyContacts">
        <property name="focusPolicy">
         <enum>Qt::NoFocus</enum>
        </property>
        <property name="toolTip">
         <string>Agenda de contactos</string>
        </property>
        <property name="statusTip">
         <string>Agenda de contactos</string>
        </property>
        <property name="whatsThis">
         <string>Agenda de contactos</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="../resources/resources.qrc">
          <normaloff>:/generales/images/addressbook.png</normaloff>:/generales/images/addressbook.png</iconset>
        </property>
        <property name="checkable">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="label_5">
        <property name="text">
         <string>Asunto</string>
        </property>
       </widget>
      </item>
      <item row="3" column="2" colspan="2">
       <widget class="QLineEdit" name="txtSubject"/>
      </item>
      <item row="1" column="2">
       <widget class="QLineEdit" name="txtTo"/>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="0" colspan="2">
    <widget class="QSplitter" name="splitter">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <widget class="QGroupBox" name="gbBody">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Cuerpo del mensaje</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QwwRichTextEdit" name="htmlEditor" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Minimum">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="focusPolicy">
          <enum>Qt::StrongFocus</enum>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QGroupBox" name="gbAttach">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="title">
       <string>Adjuntos</string>
      </property>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="1" column="1">
        <widget class="QPushButton" name="pbRemoveAttachment">
         <property name="text">
          <string>Eliminar Adjunto</string>
         </property>
         <property name="icon">
          <iconset resource="../resources/resources.qrc">
           <normaloff>:/generales/images/edit_remove.png</normaloff>:/generales/images/edit_remove.png</iconset>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QPushButton" name="pbAddAttachment">
         <property name="text">
          <string>Añadir Adjunto</string>
         </property>
         <property name="icon">
          <iconset resource="../resources/resources.qrc">
           <normaloff>:/generales/images/edit_add.png</normaloff>:/generales/images/edit_add.png</iconset>
         </property>
        </widget>
       </item>
       <item row="0" column="0" colspan="2">
        <widget class="QTableWidget" name="tableWidget">
         <property name="contextMenuPolicy">
          <enum>Qt::CustomContextMenu</enum>
         </property>
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="dragEnabled">
          <bool>true</bool>
         </property>
         <property name="dragDropMode">
          <enum>QAbstractItemView::DragDrop</enum>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
         <property name="selectionMode">
          <enum>QAbstractItemView::SingleSelection</enum>
         </property>
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectRows</enum>
         </property>
         <attribute name="horizontalHeaderDefaultSectionSize">
          <number>200</number>
         </attribute>
         <attribute name="horizontalHeaderMinimumSectionSize">
          <number>50</number>
         </attribute>
         <column>
          <property name="text">
           <string>Fichero</string>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
         </column>
         <column>
          <property name="text">
           <string>Tipo</string>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
         </column>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="pbSend">
     <property name="text">
      <string>Enviar</string>
     </property>
     <property name="icon">
      <iconset resource="../resources/resources.qrc">
       <normaloff>:/generales/images/email.png</normaloff>:/generales/images/email.png</iconset>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="pbClose">
     <property name="text">
      <string>Salir</string>
     </property>
     <property name="icon">
      <iconset resource="../resources/resources.qrc">
       <normaloff>:/generales/images/close.png</normaloff>:/generales/images/close.png</iconset>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QwwRichTextEdit</class>
   <extends>QWidget</extends>
   <header>qwwrichtextedit.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>txtTo</tabstop>
  <tabstop>txtCopy</tabstop>
  <tabstop>txtSubject</tabstop>
  <tabstop>htmlEditor</tabstop>
  <tabstop>tableWidget</tabstop>
  <tabstop>pbAddAttachment</tabstop>
  <tabstop>pbRemoveAttachment</tabstop>
  <tabstop>pbSend</tabstop>
  <tabstop>pbClose</tabstop>
  <tabstop>txtFrom</tabstop>
 </tabstops>
 <resources>
  <include location="../resources/resources.qrc"/>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
